com.unity.cinemachine/Editor/Samples/SampleDependencyImporter.cs

```diff
@@ -170,6 +182,32 @@ static bool PromptUserImportDependencyConfirmation(List<string> dependencies)
                     "Import samples without their dependencies");
             }
         }
+
+        void ConvertMaterials(string assetPath)
+        {
+            if (m_SampleConfiguration != null)
+            {
+                if (assetPath.EndsWith(".mat") && !assetPath.EndsWith("FadeOut.mat") && !m_UpgradedMaterials.Contains(assetPath))
```

Comment:
How do you feel about this implementation?  Can you think of other ways to exclude the FadeOut material from auto-upgrading?  Are any of them better than this one?

Reply:
Yes, I'll fix that.